plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/purity/PureOperatorWhitelist.kt

```diff
@@ -0,0 +1,27 @@
+package org.jetbrains.kotlin.formver.purity
+
+/**
+ * Utility that decides whether a *mangled* method name represents
+ * a primitive operator we always treat as pure.
+ */
+object PureOperatorWhitelist {
+
+    /** Substrings that are guaranteed to appear in the mangled spelling
+     *  of the corresponding operator functions. Extend as needed. */
+    private val TOKENS: Set<String> = setOf(
+        // unary
+        "unaryMinus", "unaryPlus", "not",
+        // binary arithmetic & bitwise
+        "plus", "minus", "times", "div", "rem",
```

Comment:
You can remove division and remainder, since our implementations of those currently aren't pure.